qiskit_experiments/calibration_management/calibrations.py

```diff
@@ -52,20 +56,55 @@ class Calibrations:
     ScheduleBlock are supported.
     """
 
+    # The name of the parameter under which the qubit frequencies are registered.
+    __qubit_freq_parameter__ = "qubit_lo_freq"
+
+    # The name of the parameter under which the readout frequencies are registered.
+    __readout_freq_parameter__ = "meas_lo_freq"
+
     # The channel indices need to be parameterized following this regex.
     __channel_pattern__ = r"^ch\d[\.\d]*\${0,1}[\d]*$"
 
-    def __init__(self, control_config: Dict[Tuple[int, ...], List[ControlChannel]] = None):
+    def __init__(
+        self,
+        coupling_map: Optional[List[List[int]]] = None,
+        control_channel_map: Optional[Dict[Tuple[int, ...], List[ControlChannel]]] = None,
+        library: Optional[BasisGateLibrary] = None,
```

Comment:
```suggestion
        library: Optional[Union[BasisGateLibrary, List[BasisGateLibrary]] = None,
```
? to stabilize interface. It should be okey raising an error (not implemented) for now.

Reply:
Sounds okay. e4d1c0c